flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/extraction/FunctionTemplate.java

```diff
@@ -74,6 +74,28 @@ static FunctionTemplate fromAnnotation(DataTypeFactory typeFactory, FunctionHint
 		);
 	}
 
+	/**
```

Comment:
This is just moving the code, no other changes, right?

Reply:
it was moved because the visibility changed